docs/faq.rst

```diff
@@ -187,6 +187,38 @@ all the part files and manually concatenate them together.
 
 For more details, see :ref:`checkpoints_parts`.
 
+My repository is corrupt, how can I restore from an older but working copy?
+---------------------------------------------------------------------------
+
+The recommended method is to delete the corrupted repository, and then copy the
+working repo to the same location. The delete command will completely remove the
+corrupt repo and delete the corresponding cache and security subdirectory in
+``~/.config/borg/security``, including the nonce value.
+
+If you use the repo copy for creating new archives, Borg will initialize a fresh
+nonce which is bad for security reasons (nonce values should never be reused).
+To prevent this, the security subdirectory should be saved before deleting, and
+later moved back into place.
+
+Example:
+
+::
+
+    # Get the repo ID from repo config.
+    REPO_ID=$(grep 'id = ' /path/to/repo-good/config | cut -d ' ' -f 3)
```

Comment:
don't we have some borg command showing the id? that would also work for remote ssh: repos.

Reply:
Oh yes, it's ``borg config /path/to/repo id``. I will use that.